cg/store/models.py

```diff
@@ -393,6 +393,7 @@ class Case(Model, PriorityMixin):
     internal_id = Column(types.String(32), unique=True, nullable=False)
     is_compressible = Column(types.Boolean, nullable=False, default=True)
     name = Column(types.String(128), nullable=False)
+    order_id = Column(ForeignKey("order.id", ondelete="CASCADE"))
     ordered_at = Column(types.DateTime, default=dt.datetime.now)
```

Comment:
is the case.ordered_at redundant when we have an order table that has the order date?

Reply:
Most likely, but all logic which reroutes functionality will be in other PRs.